README.md

```diff
@@ -59,12 +59,9 @@ $ vagrant up
 
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
 
-The flask app has now been altered so that it is now partially integrated to the Trello Website API. To run the altered app one needs:
-
--A Trello Account to access the Trello API (This is free)
-
--A .env file with the Trello Enviromental variables (Key, Token, List IDs etc) inside, as well as a Flask Secret Key as well as the server configuration
+The flask app has now been altered so that it is now integrated to the MongoDB Server.
 
+In order to run the app one needs to have access to a MongoDB Client and to ensure one has the correct Mongo Client connection string in their environment file. 
```

Comment:
The reader currently has to read the code to work out what the necessary environment variables are. 

If you don't document them in the README, you should at least put them in .env.template